tests/src/Functional/ConfigFormTest.php

```diff
@@ -56,13 +56,16 @@ public function testConfigForm(): void {
 
     // The form contains default values from module install.
     $this->drupalGet(Url::fromRoute('collabora-online.settings'));
-    $assert_session->fieldValueEquals('Collabora Online server URL', 'https://localhost:9980/');
-    $assert_session->fieldValueEquals('WOPI host URL', 'https://localhost/');
-    $assert_session->fieldValueEquals('JWT private key', '');
-    $assert_session->fieldValueEquals('Access Token Expiration (in seconds)', '86400');
-    $assert_session->checkboxNotChecked('Disable TLS certificate check for COOL.');
-    $assert_session->checkboxChecked('Verify proof header and timestamp in incoming WOPI requests.');
-    $assert_session->checkboxChecked('Allow COOL to use fullscreen mode.');
+    $this->assertFormValues(
+      'https://localhost:9980/',
+      'https://localhost/',
+      '',
+      '86400',
+      FALSE,
+      TRUE,
+      TRUE,
+      FALSE,
+    );
```

Comment:
Overall the class becomes longer this way (due to more methods), but imo this main method becomes more readable.